features/step_definitions/command_steps.rb

```diff
@@ -231,7 +231,8 @@
   $minion.run('sysctl net.ipv6.conf.all.forwarding=1')
 end
 
-And(/^I register the centos7 as tradclient$/) do
+And(/^I register the "([^"])" as tradclient$/) do |arg1|
+  node = get_target(arg1)
```

Comment:
here don't use `arg1` as argument. Use proper names like `system`  or name variable that give idea what the var is